docs/Design-and-Modeling/Configure-Spectral.md

```diff
@@ -0,0 +1,13 @@
+# Configure Spectral Rules 
+
+![Configure Spectral](../assets/images/spectral-config.png)
+
+## What 
+Spectral, Stoplight’s linting tool, can be configured to create custom rulesets to match your organization's style guides. Enable or disable rules, create new rules, change their severity, or add new descriptions.
+
+## How 
+1. Select a **project** 
+2. Click **+** and select **File** from the dropdown 
+3. Input ```.spectral.yml``` for the file name 
```

Comment:
If I could suggest something, let's recommend usage of `json` rather than `yaml` whenever possible unless there is a strong reason we recommend YAML that I'm unaware of?
```suggestion
3. Input ```.spectral.json``` for the file name 
```
In general, JSON is a more developer-friendly and standardized format than YAML, aaand autocompletion for JSON files will always be more reliable no matter what, therefore I'd really encourage everyone to use JSON if possible.

@rossmcdonald @marbemac @lottamus thoughts?

Reply:
We should reference that it can be done in either, but pushing towards JSON is fine. The hard part is that all of the Spectral ruleset docs are in YAML, so once they click the link going to the ruleset docs they have to translate to JSON. Not a big deal at the moment, though.